v8/pretty_breadcrumbs/.gitignore

```diff
@@ -0,0 +1 @@
+__pycache__
```

Comment:
should not be necessary, we have that in a global .gitignore for this repo

Reply:
Ok